src/routes/tutorial/[slug]/filetree/Item.svelte

```diff
@@ -188,24 +188,7 @@
 	}
 
 	[aria-current='true'] {
-		color: var(--prime);
+		color: hsl(15, 100%, 50%);
 	}
 
-	[aria-current='true']::after {
-		content: '';
-		position: absolute;
-		width: 1rem;
-		height: 1rem;
-		top: 0.3rem;
-		right: calc(-0.6rem - 2px);
-		background-color: var(--sk-back-3);
-		border: 1px solid var(--sk-back-4);
-		transform: translate(0, 0.2rem) rotate(45deg);
-		z-index: 2;
-	}
-
-	[aria-current='true']:has(:focus-visible)::after,
-	:global(.mobile-filetree) [aria-current='true']::after {
-		display: none;
-	}
```

Comment:
We shouldn't remove this - this adds the indented caret next to the selected file 
<img width="202" alt="image" src="https://github.com/sveltejs/learn.svelte.dev/assets/4992896/d6d40ce3-0cfc-4bdb-be7b-64f459e3905e">

Reply:
I have restored this section, but I believe the small arrow here is unnecessary. It is similar to making a menu item both bold and colorized.